index.html

```diff
@@ -17,63 +17,74 @@
             <li><a href='#'>Contact</a></li>
+            <li><a href='#'><img src="img/Union-nav.png"></a></li>
           </ul>
-        </div>
-      </div>
+      
+    
     </nav>
   </header>
   <section class="main">
-    <h1>Hey There.I’m Alex</h1>
+    <div class="main-header">
+    <h1>Hey There.<br>I’m Alex</h1>
     <h3>I’am a Software Developer</h3>
     <p>I can help you build a product , feature or website Look through some of my work and experience!
       If you like what you see and have a project you need coded,
       don’t hestiate to contact me.</p>
-
+    </div>
+    <div class="soaial-wriper">
     <ul class="social-icon">
       <li class="social"><a href="#"><img src="img\github.png"></a></li>
       <li class="social"><a href="#"><img src="img\linkedin.png"></a></li>
       <li class="social"><a href="#"><img src="img\Vector2.png"></a></li>
       <li class="social"><a href="#"><img src="img\twitter.png"></a></li>
       <li class="social"><a href="#"><img src="img\medium.png"></a></li>
     </ul>
+    </div>
   </section>
 
   <!--Second section-MY WORK-->
   <section class="middle">
-    <h1 class="headline">My recent works</h1>
+    <div class="headline">
+    <h1 >My recent works</h1>
     <hr>
-
-    <div class="grid-container">
-      <!--Card 1-->
-      <div class="first-card-grid-item flex-container">
-        <div class="first-card-flex-item">
-          <img src="img\card1-Placeholder.png">
-          <h1>Multi-Post Stories</h1>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-            industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-          <ul class="li-cards">
-            <li class="lang-card1">css</li>
-            <li class="lang-card1">html</li>
-            <li class="lang-card1">bootstrap</li>
-            <li class="lang-card1">Ruby</li>
-          </ul>
-          <button class="card1-btn">See Project</button>
+    </div>
+           <!--Card 1-->
+        <div class="first-card-grid-item flex-container">
+          <div class="first-card-flex-item">
+              <div class="img-middle">
+            <img src="img\card1-Placeholder.png">
+              </div>
+              <div class="first-flex">
+            <h1>Multi-Post Stories</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+              industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="li-cards">
+              <li class="lang-card1">css</li>
+              <li class="lang-card1">html</li>
+              <li class="lang-card1">bootstrap</li>
+              <li class="lang-card1">Ruby</li>
+            </ul>
+            <button class="card1-btn">See Project</button>
+              </div>
+          </div>
         </div>
```

Comment:
Kindly make sure the projects section is also designed according to the specifications of the figma design. Please check all dimensions with regards to the width, height, font-size, line-height, margins, paddings, etc and make it conform to that of the figma design. At the moment there are a lot of disparities as shown in the images below. 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 21-05-46](https://user-images.githubusercontent.com/65233860/193142660-7704d600-328c-42d4-96c8-216fe0d4fca7.png)  |  ![Screenshot from 2022-09-29 21-06-03](https://user-images.githubusercontent.com/65233860/193142678-38324a8e-a790-4682-b16f-451f48059f21.png)